packages/components/src/Button/__snapshots__/button.spec.tsx.snap

```diff
@@ -1070,6 +1070,6 @@ exports[`<Button /> passes test ids down properly 1`] = `
   data-testid="example-test-id"
   type="button"
 >
-  Button with data-testid
+  Button
```

Comment:
These snapshot tests are now using stories, instead of directly rendering the component.